controllers/managedocs_controller.go

```diff
@@ -1786,3 +1795,30 @@ func (r *ManagedOCSReconciler) setDeviceSetCount(deviceSet *ocsv1.StorageDeviceS
 		deviceSet.Count = currDeviceSetCount
 	}
 }
+
+func (r *ManagedOCSReconciler) reconcileRookOverrideConfig() error {
+	if r.DeploymentType != consumerDeploymentType {
```

Comment:
I believe this change needs to go on provider deployment, not consumer deployments.

Reply:
Done